File: JavaScript/fizzBuzz.js
const min = process.argv[2];
const max = process.argv[3];

for (let i = min; i <= max; i++) {
  if ((i % 3 == 0) & (i % 5 == 0)) console.log(i + " FizzBuzz");
  else if (i % 3 == 0) {
    console.log(i + " Fizz");
  } else if (i % 5 == 0) {
    console.log(i + " Buzz");
  }
}

File: src/com/company/FizzBuzz.java
package com.company;

public class FizzBuzz {

    public void runFizzBuzz(int min, int max) {
        for (int i = min; i <= max; i++) {
            if (i % 3 == 0 & i % 5 == 0) System.out.println(i + " FizzBuzz");
            else if (i % 3 == 0){
                System.out.println(i + " Fizz");
            } else if (i % 5 == 0) {
                System.out.println(i + " Buzz");
            }
        }
    }
}
